docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/hetero-execution.rst

```diff
@@ -90,9 +100,33 @@ It does not take into account device peculiarities such as the inability to infe
          :language: cpp
          :fragment: [compile_model]
 
+Pipeline parallelism
+------------------------
+
+The pipeline parallelism is set via ``ov::hint::model_distribution_policy``, This mode is an efficient technique to infer large models on multiple devices. The model is split into multiple stages, and each stage is assigned to a different device (``dGPU``, ``iGPU``, ``CPU``, etc.). This mode assign operations to different devices as reasonably as possible, ensuring that different stages can be executed in sequence and minimizing the amount of data transfer between different devices.
+
+For large models which don’t fit on a single first priority device, model pipeline parallelism is employed where certain parts of the model are placed on different devices to ensure that the device has enough memory to infer these operations, and assign other operations to next priority device.
+
```

Comment:
maybe can re-organize this part? kind of confusing of **different devices, and next priority device**

Reply:
Yes, the description that may cause confusion has been removed.